setup.py

```diff
@@ -15,7 +15,7 @@
                  "Intended Audience :: Developers",
                  "License :: OSI Approved"],
     license='BSD-2-CLAUSE',
-    version='1.1.5',
+    version='1.1.7',
```

Comment:
Not an issue but I saw this version needs to be updated constantly. Maybe we can consider automating it later.

Reply:
Yes it'll be nice to have it automated. There are solutions but haven't got a chance to pull in.